lms/services/canvas.py

```diff
@@ -0,0 +1,43 @@
+from lms.services.exceptions import CanvasFileNotFoundInCourse
+
+
+class CanvasService:
+    """A high level Canvas service."""
+
+    def __init__(self, db_session, canvas_api):
+        self._db_session = db_session
```

Comment:
Is the DB accessed from this service?

Reply:
No not yet... This is actually spill over from future PRs. It's a good spot, but I'll leave it in as I'll just have to rewrite the tests to take it out and then put it back later.